clients/fides-js/package.json

```diff
@@ -23,13 +23,26 @@
     "directory": "clients/fides-js"
   },
   "dependencies": {
+    "eslint-config-preact": "^1.3.0",
+    "npm": "^9.6.5",
+    "preact": "^10.13.2",
```

Comment:
Note that only `preact` should be here - the others are dev dependencies. I fixed that 👍